.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "riscv-binutils-gdb"]
+	path = riscv-binutils-gdb
+	url = https://github.com/riscv/riscv-binutils-gdb.git
```

Comment:
It even keeps the same URL scheme, so if you've cloned the root repo over ssh+git then your modules will be too.  Saves adding the proper origin in each submodule.